Core/StateManager.cs

```diff
@@ -1,24 +1,25 @@
-﻿using Newtonsoft.Json;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.IO;
+using Newtonsoft.Json;
 
 namespace Core
 {
-    public class StateManager
-    {
-        public static SortedSet<Transaction> Load()
-        {
-            if (!File.Exists("state.json"))
-            {
-                return new SortedSet<Transaction>();
-            }
-            return JsonConvert.DeserializeObject<SortedSet<Transaction>>(File.ReadAllText("state.json"));
-        }
+	public class StateManager
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931949)